### PR TITLE
Send HDF map to distributed alternatives provider

### PR DESCRIPTION
**Pull request by Kobzol**
This PR integrates the HDF map with the distributed alternatives provider implemented in [evkit](https://code.it4i.cz/everest/evkit).

### CODE REVIEW

File: ruth/simulator/kernels.py [hunk 1]
```diff
@@ -31,22 +42,41 @@ def __init__(self, port: int):
...
+        messages = [Message(kind="alternatives", data={
+            "start": map.osm_to_hdf5_id(v.next_routing_od_nodes[0]),
+            "destination": map.osm_to_hdf5_id(v.next_routing_od_nodes[1]),
+            "max_routes": k
+        }) for v in vehicles]
+        # logging.info(f"Sending alternatives to distributed worker: {messages}")
+        results = self.client.compute(messages)
+        # logging.info(f"Response from worker: {results}")
+        remapped_routes = [
+            [[map.hdf5_to_osm_id(segment) for segment in route] for route in result["routes"]]
```
Comment (frackledust) [inline ruth/simulator/kernels.py:63]: Here segment should probably be named node_id instead.

Comment (Kobzol) [reply ruth/simulator/kernels.py:63]: Is there something like an explicit segment ID (in the OSMNX maps or HDF5)? The code mentions it sometimes. Or do only nodes have IDs and "segment ID" is always just `node ID start - node ID end`?

Comment (frackledust) [reply ruth/simulator/kernels.py:63]: Yes, there is a segment ID, but the map doesn't use it for indexing, just in the segment data info section. The segments are indexed by start and end nodes.

Comment (Kobzol) [reply ruth/simulator/kernels.py]: Ok, changed the naming :)

**Comment by Kobzol** [comment]
Btw, if you click on the "force-pushed" text, you can see what has changed if a force-push has happened:
image

**Review by frackledust — APPROVED** [review]

**Pull request merged**